src/OrchardCore/Orchard.Mvc.Core/Extensions/ModularServiceCollectionExtensions.cs

```diff
@@ -62,6 +61,13 @@ public static class ModularServiceCollectionExtensions
             return services;
         }
 
+        public static void AddTagHelpers(this IServiceProvider serviceProvider, string assemblyName)
```

Comment:
The `Type` based one is very confusing. You could think you are adding a single type where you are actually adding the full asembly.